test/integration/default/serverspec/default_spec.rb

```diff
@@ -109,4 +109,21 @@
     its(:stdout) { should match /tag: Pulling from new/ }
     its(:stdout) { should match /Image is up to date/ }
   end
+
+  describe 'docker-gc recipe' do
+
```

Comment:
There should be no newline here. Usually newlines are placed between "end" and "describe"